latest/PIT/pit_common.yaml

```
@@ -0,0 +1,188 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_common-1.0.0
+
+title: All PIT common metadata
+description: |
+  This schema is the union of all Roman PIT common metadata, including
+  those found in the referenced pit_basic schema.
+type: object
+allOf:
+  - $ref: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_basic-1.0.0
+  - type: object
+    properties:
+      file_date:
+        title: File creation date
+        description: |
+          File creation date-time in ISO 8601 format with UTC time
+          scale.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      start_time:
+        title: Start time
+        description: |
+          Start time of data acquisition in ISO 8601 format.
+          tag: tag:stsci.edu:asdf/time/time-1.*
+      end_time:
+        title: End time
+        description: |
+          End time of data acquisition in ISO 8601 format.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      exposure_time:
+        # Optional
+        title: Exposure time (s)
+        description: |
+          The amount of time elapsed between the start and the end of
+          the observation or data stream.
+        type: number
+        unit: "s"
+      effective_exposure_time:
+        title: Effective exposure time (s)
+        description: |
+          The total accumulated effective exposure duration for this
+          data product, excluding dead time and lost time.
+        type: number
+        unit: "s"
+      instrument:
+        title: Instrument observing configuration
+        type: object
+        properties:
+          name:
+            title: Instrument name
+            anyOf:
+              - type: string
+              - type: "null"
+          detector:
+            title: Detector
+            anyOf:
+              - type: string
+              - type: "null"
+          optical_element:
+            title: Optical element
+            description: |
+              Name of filter, disperser, or other optical element.
+            anyOf:
+              - type: string
+              - type: "null"
+        required: [name, detector, optical_element]
+      telescope:
+        title: Telescope name
+        description: |
+          Name of telescope, typically Roman.
+        anyOf:
+          - type: string
+          - type: "null"
+      target_coordinates:
+        title: Position and region for this product
+        type: object
+        properties:
+          reference_frame:
+            anyOf:
+              - type: string
+              - type: "null"
+          ra:
+            title: Right ascension (deg)
+            description: |
+              Characteristic right ascension in degrees; typically the
+              coordinate of the source of interest, or the center of
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
```

```suggestion
              Characteristic right ascension in degrees; typically the
              coordinate of the source of interest, or the center of
              the spatial image. May be null for large survey images,
              simulated data, etc.
```
Can we refine this to remove the ambiguity about "center" or "source of interest"? For what files do we expect that "source of interest" will be used?

---

I'd prefer not to be overly prescriptive but I'm happy to at least change this to just say "typically the center of the spatial image.", if that's good enough.

(This is just the coordinates that would go in CAOM (which powers the Portal and astroquery.Observations), where the field is rather loosely defined as "the position of the observation". Affects certain advanced search fields, but not cone searches, which use sky footprints.)